docs/modules/indexes/document_loaders.rst

```diff
@@ -40,10 +40,12 @@ For detailed instructions on how to get set up with Unstructured, see installati
    ./document_loaders/examples/file_directory.ipynb
    ./document_loaders/examples/html.ipynb
    ./document_loaders/examples/image.ipynb
-   ./document_loaders/examples/jupyter_notebook.ipynb
+   ./document_loaders/examples/image.ipynb
```

Comment:
This one looks duplicated (image.ipynb above)

Reply:
Good catch. Thanks!